packages/vue/src/components/NavDesktopEdu/NavDesktopEdu.vue

```diff
@@ -12,7 +12,7 @@
     }"
   >
     <!-- navbar -->
-    <div class="header-bg z-10 max-w-screen-3xl absolute inset-0 mx-auto"></div>
+    <div class="header-bg z-10 absolute inset-0 mx-auto"></div>
```

Comment:
1. I think this class also needs to be removed from the beta banner.
2. Any reason to not also remove it from the WWW desktop nav?

Reply:
1. Yes, plan to do that in www PR that also updates the explorer-1 version
2. Yes, due to the nature of the nav sometimes being transparent with a gradient, and sometimes solid white.